Gopkg.lock

```diff
@@ -120,6 +120,12 @@
   packages = ["unix","windows"]
   revision = "fb4cac33e3196ff7f507ab9b2d2a44b0142f5b5a"
 
+[[projects]]
+  branch = "v2"
+  name = "gopkg.in/yaml.v2"
+  packages = ["."]
+  revision = "cd8b52f8269e0feb286dfeef29f8fe4d5b397e0b"
+
```

Comment:
Realising now how tiny the change is to make `cc-env` output TOML format data instead, I think it would be churlish not to do this.

I'm still much prefer YAML, but -- as you say -- we already have TOML support. One nice touch that TOML has over YAML is that the output is broken into paragraphs ((whereas the YAML gets dumped as a single block of text).